Source/Core/CesiumIon.js

```diff
@@ -0,0 +1,142 @@
+define([
+        './Check',
+        './Credit',
+        './defaultValue',
+        './defined',
+        './Resource',
+        './RuntimeError',
+        './CesiumIonResource'
+    ], function(
+        Check,
+        Credit,
+        defaultValue,
+        defined,
+        Resource,
+        RuntimeError,
+        CesiumIonResource) {
+    'use strict';
+
+    /**
+     * Utility object for working with the Cesium ion API.
+     *
+     * @see https://cesium.com
+     *
+     * @exports CesiumIon
```

Comment:
Were we going to have this export `Ion` so it's `Cesium.Ion` instead of `Cesium.CesiumIon`?  I thought I read that somewhere but maybe I'm misremembering.

Reply:
Read number 2 in my PR description.